hochschulfaechersystematik.ttl

```diff
@@ -1640,7 +1640,7 @@
   skos:broader <n37> ;
   skos:notation "237" ;
   skos:deprecated true ;
```

Comment:
instead of skos:deprecated it should be owl:deprecated (at least that is what we discussed in https://github.com/skohub-io/skohub-vocabs/issues/287

Reply:
Ja, du hast recht. Vor allem, weil es `skos:deprecated` gar nicht gibt. Wir haben alle eine Menge gelernt bei diese Sache...